simple_history/admin.py

```diff
@@ -84,6 +84,7 @@ def history_view(self, request, object_id, extra_context=None):
             'admin_user_view': admin_user_view,
             'history_list_display': history_list_display,
         }
+        context.update(self.admin_site.each_context(request))
```

Comment:
Looks good, but why not include in the initial context definition? Like https://github.com/django/django/blob/master/django/contrib/admin/options.py#L1608

Reply:
That syntax is not supported for python2.7.